src/collections/_documentation/platforms/javascript/sourcemaps/availability.md

```diff
@@ -0,0 +1,138 @@
+---
+title: 'Hosting / Uploading'
+sidebar_order: 2
+---
+
+<!-- ## Making Source Maps Available to Sentry -->
```

Comment:
Do we want to keep this as a heading? Otherwise, might as well remove.

Reply:
oh yeah woops